### PR TITLE
Add mock-up of champions page

### LINKED ISSUE #382: Add Champions Program project
We can use the same scheme as the review project.  The first section will be:

General information about the program
Benefit
TimeLine

Then we will add: who to apply.

After the application close, we will add a section with:

- Our Champions (like our team in the other sections)
- Our Mentors (like our team in the other sections)

### PR DESCRIPTION
**Pull request by maelle**
Fix #382

### CODE REVIEW

**Comment by maelle** [comment]
https://deploy-preview-383--ropensci.netlify.app/

**Comment by maelle** [comment]
Please ping me if any help is needed here :smile_cat:

**Comment by yabellini** [comment]
I need to add a template of the form. I create a .md and I put it inside the champion folder, but this creates a blank item in the project menu and I don't know how to remove that blank item.  Is there a specific place to put the template file to avoid this?

**Comment by maelle** [comment]
@yabellini I fixed that (it was a problem on the Hugo side, which means my fault :wink: ).

I furthermore increased the margin around images. Does that look ok for you?

File: content/commcalls/2022-10-24/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "rOpenSci Champions Program"
```
Comment (maelle) [inline content/commcalls/2022-10-24/index.md:2]: I've added a link to it but the co-working event hasn't been created yet, correct?

Comment (yabellini) [reply content/commcalls/2022-10-24/index.md:2]: The co-working is this link: /events/coworking-2022-11/
The comm call is in this PR, should I create before?

File: content/champions/_index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    - how to plan and facilitate engaging and inclusive workshops to support participants’ success,
+    - knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects, and
+    - technical skills in software development and review.
+
+* Personal and group mentorship:
+
+    - A series of meetings with mentors;
+    - Quarterly cohort calls.
+
+See [the program time line for more detail](#Tentative-Program-timeline).
```
Comment (yabellini) [inline content/champions/_index.md:57]: @maelle this internal link doesn't work on the preview (or in my local build, and I don't know why :-( ).  Can you check what I'm doing wrong here?

File: content/champions/_index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+In its inaugural year, the _rOpenSci Champions Program_ aims to support R developers, from around the world, and who identify themselves as members of a group that is systematically excluded from the open software community to:
+
+* Gain the skills to develop their own R package or;
+* Gain confidence in submitting an R package for peer review or become a reviewer.
+* Learn how to lead a workshop or other event about package development or software peer review -  or another rOpenSci activity - in their local community.
+* Connect with a supportive network of peers and mentors within the R ecosystem.
+* Provide feedback to help rOpenSci to continue to host relevant, welcoming activities for R developers.
+* Gain transferable skills and experiences that are sought after for careers in open source software.
+
+
```
Comment (yabellini) [inline content/champions/_index.md:24]:
```suggestion
We are looking six to nine Champions for this first pilot. We expect to co-create the rOpenSci Champion program with the selected applicants and influence future editions with their experience and feedback.

We look forward to receiving many applications and are committed to directing non-selected applicants towards other ways in which they can contribute to rOpenSci.


```

File: content/commcalls/2022-10-24/index.md (hunk 1 above)
Comment (maelle) [reply content/commcalls/2022-10-24/index.md:2]: it's fine. can you add this link to the table of the program?

File: content/champions/_index.md (hunk 2 above)
Comment (maelle) [reply content/champions/_index.md:61]: it's case sensitive so use `#tentative-program-timeline`

File: content/champions/_index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+|Jun 2023|Cohort meeting: Reflections on projects|Champions will be invited to share takeaways from their projects with the cohort.|1.5 h|
+|August - November 2023|Champions outreach: host events, deliver talk, write about their experiences.|Champion's host: a workshop on good practices to build packages or good practices in software peer review or deliver a talk on a conference/event and one community event to gather feedback to improve rOpenSci’s projects|1 to 3 h|
+|September 2023|Cohort meeting: Reflections on events |Champions will be invited to share takeaways from their events with the cohort.|1.5 h|
+|December 2023|Cohort meeting: Reflections on program|Champions will be invited to share takeaways from the program with the cohort and start to define future roles for Champions.|1.5 h|
+|February 2024|End of term|Close cohort and define new roles for Champions.|1 h|
+
+
+## How to apply 
+
+Please, [fill out the application form](https://airtable.com/shrAsYlSXU0coJ5Ld) (this [template shows the information we will require](/files/champions_template)) before __Friday Nov 7, 2022.__
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/champions/_index.md]: should now be fixed, sorry

File: content/champions/_index.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    - how to plan and facilitate engaging and inclusive workshops to support participants’ success,
+    - knowledge of various channels through which new members can engage in and contribute to rOpenSci and R projects, and
+    - technical skills in software development and review.
+
+* Personal and group mentorship:
+
+    - A series of meetings with mentors;
+    - Quarterly cohort calls.
+
+See [the program time line for more detail](#Tentative-Program-timeline).
```
Comment (yabellini) [inline content/champions/_index.md:61]:
```suggestion
See [the program time line for more detail](#tentative-program-timeline).
```

File: content/champions/_index.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+Please, [fill out the application form](https://airtable.com/shrAsYlSXU0coJ5Ld) (this [template shows the information we will require](/champions/files/champions_template)) before __Friday Nov 7, 2022.__
+The application forms __have to be filled in English__.
+
+### Application Timeline
+
+|Date|Event|
+|----|-----|
+|Sep 23, 2022|Call for Application opens|
+|Oct 24, 2022|[OPTIONAL] [Community Call - Tips for submitting your Champions Program application](/commcalls/oct2022-champions/)|
+|Nov 1, 2022|[OPTIONAL] Co-working hours: receive help with your application to the Champions Program.|
```
Comment (yabellini) [inline content/champions/_index.md:137]:
```suggestion
|Nov 1, 2022|[OPTIONAL] [Co-working hours](/events/coworking-2022-11/): receive help with your application to the Champions Program.|
```

**Comment by yabellini** [comment]
Thanks a lot, @maelle !!! <3

**Pull request merged**